lib/src/ovis_auth/auth.c

```diff
@@ -141,8 +142,10 @@ char *ovis_auth_get_secretword(const char *path, ovis_auth_log_fn_t log)
 	struct stat pstat;
 	if (stat(path, &pstat)) {
 		ret = errno;
+		char ebuf[80];
+		strerror_r(errno, ebuf, sizeof(ebuf));
 		log("Auth error: %s while trying to stat %s\n",
-			strerror(errno), path);
+			ebuf, path);
```

Comment:
Why convert this to strerror_r() when the rest of this commit is changing everything to ovis_strerror()?

Reply:
to at least have it be thread safe and the log messages not potentially corrupted. switching to ovis_strerror would be my preference.